Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/PIDDrive.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc.team199.Robot2018.commands;
+
+import org.usfirst.frc.team199.Robot2018.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class PIDDrive extends Command {
```

Comment:
Need a comment describing what this command is supposed to do. My guess is that it is intended to use pid to drive a target distance straight forward. However, it doesn't look like it will do that since it seems to always send the same value to both motors. Mechanical and electrical differences between the left and right side of the drivetrain will cause them not to move at the same speed for the same value, and the robot will drift left or right.

Reply:
Drivetrain implements PIDOutput (hopefully correctly) so the value does change. Differences can/should be accounted for by the PID constants.

Reply:
The value changes over time due to Drivetrain's PID code, but at any given time the same value is sent directly to both motor controllers. What am I missing?

Reply:
I don't currently have a "correct" fix to the proposed issue; more research would be required. One possible fix could be adding coefficients in the Drivetrain class so that callers can assume that both sides run equally.

Reply:
Moved to issue #28